src/diffusers/models/embeddings.py

```diff
@@ -959,7 +959,12 @@ def forward(self, ids: torch.Tensor) -> torch.Tensor:
         freqs_dtype = torch.float32 if is_mps else torch.float64
         for i in range(n_axes):
             cos, sin = get_1d_rotary_pos_embed(
-                self.axes_dim[i], pos[:, i], repeat_interleave_real=True, use_real=True, freqs_dtype=freqs_dtype
+                self.axes_dim[i],
+                pos[:, i],
+                self.theta,
```

Comment:
```suggestion
                theta=self.theta,
```
Just to match positional/keyword arguments in `get_1d_rotary_pos_embed` signature.

Reply:
okay